pkgs/by-name/bu/buffybox/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  fetchFromGitLab,
+  inih,
+  lib,
+  libdrm,
+  libinput,
+  libxkbcommon,
+  meson,
+  ninja,
+  pkg-config,
+  scdoc,
+  stdenv,
+  unstableGitUpdater,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "buffybox";
+  version = "3.2.0-unstable-2024-11-10";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.postmarketos.org";
+    owner = "postmarketOS";
+    repo = "buffybox";
+    fetchSubmodules = true; # to use its vendored lvgl
+    rev = "07e324c17564cb9aab573259a8e0824a6806a751";
+    hash = "sha256-JY9WqtRjDsQf1UVFnM6oTwyAuhlJvrhcSNJdEZ0zIus=";
+  };
+
+  depsBuildBuild = [
+    pkg-config
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    scdoc
+  ];
+
+  buildInputs = [
+    inih
+    libdrm
+    libinput
+    libxkbcommon
+  ];
+
+  env.PKG_CONFIG_SYSTEMD_SYSTEMD_SYSTEM_UNIT_DIR = "${placeholder "out"}/lib/systemd/system";
+
+  strictDeps = true;
+
+  passthru.updateScript = unstableGitUpdater { };
+
+  meta = with lib; {
+    description = "A suite of graphical applications for the terminal";
+    homepage = "https://gitlab.postmarketos.org/postmarketOS/buffybox";
+    license = licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [ colinsane ];
+    platforms = platforms.linux;
+  };
```

Comment:
I assume that in your opinion `buffyboard` can not be considered the main program since one may also want to use `unl0kr`. Is that correct? Just making sure that you deliberately did not specify `meta.mainProgram`.

Reply:
Btw. is there any relation between `unl0kr` from this package and the `unl0kr` package, which is already in Nixpkgs?

Reply:
yes, unl0kr is now part of this suite

Reply:
in fact, if this gets merged I'll gladly remove the unl0kr package and change the service so it looks for the unl0kr binary